### PR TITLE
CIA 1017 UI plus 1.1.4 fix release updates

### PR DESCRIPTION
**Pull request by thomaswinkler**

### CODE REVIEW

**Comment by MarkJMReynolds-SAG** [comment]
@MWindel Would also need cherry-picking into 10.17 line after approval, please let me know if you want us to drive that. Thanks!

File: content/users-guide/sensor-app-bundle/bluetooth-connection.md [hunk 1]
```diff
@@ -22,3 +23,10 @@ Ensure that the device you wish to connect to is switched on and in pairing mode
 Bring the device close to your smartphone, 30cm or less is ideal.
 
 When the Bluetooth device appears in the list, tap **Pair Device** to start pairing. The Bluetooth device will then be connected to your smartphone and start sending data to {{< product-c8y-iot >}} if you are currently connected.
+
+{{< c8y-admon-info >}}
+To try connecting Bluetooth sensor devices without a physical device, {{< sensor-app >}} supports Bluetooth simulation. {{< /c8y-admon-info >}}
+
+To enable or disable simulation in {{< sensor-app >}}, follow instructions at the beginning of the section to scan for connected Bluetooth devices. While scanning, an **ENABLE SIMULATION** button is be presented to enable the simulation mode. Once enabled, {{< sensor-app >}} will present devices supporting the simulation mode. Continue from here as with real physical devices.
```
Comment (MWindel) [inline content/users-guide/sensor-app-bundle/bluetooth-connection.md:30]:
```suggestion
To enable or disable the simulation in {{< sensor-app >}}, follow the instructions at the beginning of the section to scan for connected Bluetooth devices. During the scan use the **Enable simulation** button to enable the simulation mode. Once enabled, {{< sensor-app >}} will present devices supporting the simulation mode. Continue from here as with real physical devices.
```

File: content/users-guide/sensor-app-bundle/bluetooth-connection.md [hunk 2]
```diff
@@ -22,3 +23,10 @@ Ensure that the device you wish to connect to is switched on and in pairing mode
...
+
+To disable simulation, all simulated devices must be connected to get the **DISABLE SIMULATION** button. A second option is to disable via the {{< sensor-app >}} settings. In Android the settings are available via the main menu. On iOS, find {{< sensor-app >}} in the System Preferences app, select **Connectivity** and disable **Simulate Devices**. By disabling the simulation mode, all simulated devices will be disconnected and removed.
```
Comment (MWindel) [inline content/users-guide/sensor-app-bundle/bluetooth-connection.md:32]:
```suggestion
There are two ways to disable the simulation:

 * Click **Disable simulation** to end the simulation for all connected simulated devices. 
 * End the simulation via the {{< sensor-app >}} settings. On Android the settings are available via the main menu. On iOS, find {{< sensor-app >}} in the System Preferences app, select **Connectivity** and disable **Simulate devices**. By disabling the simulation mode, you disconnect and remove all simulated devices.
```

File: content/users-guide/sensor-app-bundle/register.md [hunk 3]
```diff
@@ -13,13 +13,13 @@ For earlier versions of {{< product-c8y-iot >}}, follow the steps in [Manual reg
...
 
-    ![Cockpit application](/images/users-guide/csa/csa-connect-smartphone-right-drawer.jpg)
+    ![Cockpit application](/images/users-guide/csa/csa-connect-smartphone-right-drawer.png)
 
 2.  Follow the instructions in the wizard to step 3, ensuring that the app is installed on the smartphone.
 
     ![QR code](/images/users-guide/csa/csa-connect-smartphone-wizard-step3.png)
 
-3.  From your smartphone, launch the app and tap **Connect** in the top right corner of the screen. 
+3.  From your smartphone, launch the app and tap **Register** in the top right corner of the screen.
```
Comment (MWindel) [inline content/users-guide/sensor-app-bundle/register.md:22]: @BeateRixen Do we use "tap" here in the context of "click"?

File: content/users-guide/sensor-app-bundle/register.md [hunk 4]
```diff
@@ -34,24 +34,24 @@ QR codes not supported by the {{< sensor-app >}} are highlighted using a red reg
 
 1.  On a desktop or laptop computer, open a web browser and log in to your {{< product-c8y-iot >}} tenant. In the Device Management application, select **Devices > Registration** from the navigator.
 
-    ![Manual registration](/images/users-guide/csa/csa-device-registration.jpg)
+    ![Manual registration](/images/users-guide/csa/csa-device-registration.png)
 
-2.  From your smartphone, launch the app and tap **Connect** at the top right of the screen. When the camera opens, tap **Manual registration**.
+2.  From your smartphone, launch the app and tap **Register** at the top right of the screen. When the camera opens, tap **REGISTER MANUALLY**.
```
Comment (MWindel) [inline content/users-guide/sensor-app-bundle/register.md:39]:
```suggestion
2.  From your smartphone, launch the app and tap **Register** at the top right of the screen. When the camera opens, tap **Register manually**.
```

File: content/users-guide/sensor-app-bundle/register.md [hunk 5]
```diff
@@ -34,24 +34,24 @@ QR codes not supported by the {{< sensor-app >}} are highlighted using a red reg
...
 
-3. Back in your web browser, click **Register device** and then **General device registration**. Select a meaningful and unique device ID for your smartphone and a group to assign your device to. Click **Next** and then **Complete**. The server will notice the pending registration and wait for your smartphone to show up.
+3. Back in your web browser, click **Register device** (in the top-right of the page) and then select **General**. Choose a meaningful and unique device ID for your smartphone and a group to assign your device to. Click **Next** and then **Complete**. The server will notice the pending registration and wait for your smartphone to show up.
```
Comment (MWindel) [inline content/users-guide/sensor-app-bundle/register.md:41]:
```suggestion
3. Back in your web browser, click **Register device** at the top-right of the page and select **General**. Choose a meaningful and unique device ID for your smartphone and a group to assign your device to. Click **Next** and then **Complete**. The server will notice the pending registration and wait for your smartphone to show up.
```

File: content/users-guide/sensor-app-bundle/register.md [hunk 6]
```diff
@@ -34,24 +34,24 @@ QR codes not supported by the {{< sensor-app >}} are highlighted using a red reg
...
 
-4. On your smartphone, fill in your tenant (for example "companytenant01"), select your instance (for example "{{< domain-c8y >}}"), and enter your selected device ID. If your instance is not available in the list, you can enter it manually after clicking **Add other instance**. Click **Connect** and your device will contact the server and ask to be accepted.
+4. On your smartphone, fill in your tenant (for example "companytenant01"), select your instance (for example "{{< domain-c8y >}}"), and enter your selected device ID. If your instance is not available in the list, you can enter it manually after clicking **Add other instance**. Click **Register** and your device will contact the server and ask to be accepted.
```
Comment (MWindel) [inline content/users-guide/sensor-app-bundle/register.md:43]:
```suggestion
4. On your smartphone, fill in your tenant, for example, "companytenant01", select your instance, for example, "{{< domain-c8y >}}", and enter your selected device ID. If your instance is not available in the list, you can enter it manually if you click **Add other instance**. Click **Register** for your device to contact the server and ask to be accepted.
```

File: content/users-guide/sensor-app-bundle/register.md [hunk 7]
```diff
@@ -34,24 +34,24 @@ QR codes not supported by the {{< sensor-app >}} are highlighted using a red reg
...
-If you do not accept the device request within the required time, an according message shows up. Select **Retry** to continue with the registration process.
-{{< /c8y-admon-info >}}
-
 For further information about registering a device on the platform manually, refer to [Device Management > Connecting devices](/users-guide/device-management/#connecting-devices).
 
-If you want to disconnect from {{< product-c8y-iot >}}, tap **Disconnect** in the top right corner of the screen and approve the confirmation dialog. After that you can connect to the same or any other {{< product-c8y-iot >}} instance or tenant.
+If you want to deregister from {{< product-c8y-iot >}}, tap **Deregister** in the top right corner of the screen and approve the confirmation dialog. After that you can connect to the same or any other {{< product-c8y-iot >}} instance or tenant.
+
+{{< c8y-admon-info >}}
+Deregistering a device will automatically remove the device from {{< product-c8y-iot >}}.
```
Comment (MWindel) [inline content/users-guide/sensor-app-bundle/register.md:56]:
```suggestion
Deregistering a device automatically removes the device from {{< product-c8y-iot >}}.
```

**Comment by MarkJMReynolds-SAG** [comment]
@MWindel Thanks for your suggestions. No objections on my side so I have accepted them all.

**Review by MWindel — APPROVED** [review]

**Pull request merged**